makedoc.g

```diff
@@ -15,7 +15,16 @@ LoadPackage("AutoDoc");
 AutoDoc(rec( 
     autodoc := true,
     scaffold := rec(
-        includes := [ "Tutorials.xml", "Comments.xml" ]
+        includes := [ "Tutorials.xml", 
+                      "Comments.xml" ],
+        gapdoc_latex_options := rec( EarlyExtraPreamble := """
+            \usepackage[all]{xy} 
+            \newcommand{\bbZ} {\mathbb{Z}}
+        """ ),  
+        entities := rec( 
+            orb := "<Package>orb</Package>", 
```

Comment:
Is this orb entity used anywhere?

Reply:
Also, why do we need the `xy` package for AutoDoc? Maybe you accidentally copied the kbmag makedoc into this PR?